### DISCUSSION TITLE
Copilot suddenly stopped working in vscode.

### DISCUSSION THREAD

**Discussion by DAVIDFREITAS88** [community/community#14944] [Copilot Conversations]
It keeps loading, no error or anything like that. It just loads and nothing happens. Regardless of the complexity of the suggestion.

--------------------------------------------------------------------------------------------------------------------------------------------------

[INFO] [ghostText] [2022-04-14T13:11:49.500Z] Cancelled after extractPrompt
[INFO] [default] [2022-04-14T13:11:49.592Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-04-14T13:12:03.773Z] Breaking, no choices
[INFO] [default] [2022-04-14T13:23:58.970Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-04-14T13:23:58.978Z] Breaking, no choices
[INFO] [default] [2022-04-14T13:23:59.240Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-04-14T13:23:59.513Z] Breaking, no choices
[INFO] [default] [2022-04-14T13:23:59.790Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2022-04-14T13:24:11.460Z] Breaking, no choices

**Comment by Kek5chen** [comment]
Make sure you have the latest version installed from the Extensions:
https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/gettingstarted.md

If you have, try reinstalling the plugin, restarting VS Code and connecting GitHub back to Copilot.
